contingency/contingency-api/src/main/java/com/powsybl/contingency/json/IdentifierDeserializer.java

```diff
@@ -37,56 +37,42 @@ public NetworkElementIdentifier deserialize(JsonParser parser, DeserializationCo
         String voltageLevelId1 = null;
         String voltageLevelId2 = null;
         String contingencyId = null;
-        String version = (String) deserializationContext.getAttribute("version");
+        String version = (String) deserializationContext.getAttribute("identifierListVersion");
```

Comment:
You created a utility method `JsonUtil::getSourceVersion` for that, shouldn't you use it?

Reply:
I did not remember I'll chack that